docs/references/ic-interface-spec.md

```diff
@@ -2181,6 +2182,15 @@ The following system calls provide access to the canister's environment variable
       - `dst+size` exceeds the size of the WebAssembly memory
 
 
+-   `ic0.env_var_name_exists : (name_src: I, name_size: I) -> I`; `I ∈ {i32, i64}`
+
+    Checks if an environment variable with the given name exists.
+
+    This system call traps if:
+       - `name_src+name_size` exceeds the size of the WebAssembly memory
```

Comment:
There should also be a maximum length of the name (the concrete value omitted in the spec).